app/js/reducers/discontinueOrderReducer.js

```diff
@@ -0,0 +1,30 @@
+import {
+  DISCONTINUE_ORDER_FAILURE,
+  DISCONTINUE_ORDER_SUCCESS,
+} from '../actions/actionTypes';
+import mockData from '../../../__mocks__/mockData';
+
+const initialState = [
+  mockData.defaultpatientActiveOrder,
+  mockData.pastOrders,
+  mockData.loading,
+];
+
+export default (state = initialState, action) => {
+  switch (action.type) {
+    case DISCONTINUE_ORDER_SUCCESS:
+      return {
+        ...state,
+        activeOrders: state.activeOrders,
+        pastOrders: state.pastOrders,
+        loading: action.status,
```

Comment:
I can notice some code duplication here. state already contains activeOrders and pastOrders so I guess there is no need spreading them into the state. More like:
{
...state, loading: action.status
}